IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcCoordinateReferenceSystem/DocEntity.xml

```diff
@@ -44,6 +44,18 @@ ELLIPSOID[&amp;lt;name&amp;gt;,&amp;lt;major axis&amp;gt;,&amp;lt;inverse flatte
 		<DocAttribute Name="HasCoordinateOperation" UniqueId="83795333-6df7-46ac-b61e-5101640f8615" DefinedType="IfcCoordinateOperation" AggregationType="3" AggregationLower="0" AggregationUpper="1" Inverse="SourceCRS" XsdFormat="attribute">
 			<Documentation>Indicates conversion between coordinate systems. In particular it refers to an _IfcCoordinateOperation_ between this coordinate reference system, and another Geographic coordinate reference system.</Documentation>
 		</DocAttribute>
+		<DocAttribute Name="HasWellKnowTextRepresentation" UniqueId="9b9f3bbc-d2fc-4157-a1c7-9a86730b2575" DefinedType="IfcWellKnownText" AttributeFlags="1" Inverse="CoordinateReferenceSystem">
+			<Documentation>Well known text (string) according to ISO 19162.</Documentation>
```

Comment:
I would add a note, saying that if EPSG code is given in *Name* as well, the *Name* takes precedence.

Reply:
Agree.